zeppelin-server/src/test/java/org/apache/zeppelin/AbstractZeppelinIT.java

```diff
@@ -18,7 +18,9 @@
 package org.apache.zeppelin;
 
 
+import com.amazonaws.util.Base64;
```

Comment:
should we prefer apache.common instead? https://commons.apache.org/proper/commons-codec/apidocs/org/apache/commons/codec/binary/Base64.html